index/src/javascript/error.js

```diff
@@ -0,0 +1,185 @@
+/*
+ * Copyright (c) 2020, FinancialForce.com, inc
```

Comment:
I wouldn't go back and change any of them but I find these copyrights a little verbose and they mark the code slightly harder to read. I would go for the short version where we can.

Reply:
We should clarify with Legal, but I believe the copyright policy is different for open source repositories. I think we're supposed to include the full copyright notice with correct dates,

Reply:
Open source has to be this notice.